app/controllers/searches_controller.rb

```diff
@@ -19,7 +19,7 @@ class SearchesController < ApplicationController
   before_action :build_search_histories, except: (%i(create) + DEBUG_PAGES)
 
   before_action :set_twitter_user,       only: SEARCH_MENUS
-  before_action :create_log,             only: SEARCH_MENUS
+  before_action :create_log,             only: (%i(new create waiting menu welcome) + SEARCH_MENUS)
```

Comment:
__[rubocop]__
* Line is too long. [99/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)